### PR TITLE
Code Clean up 

### PR DESCRIPTION
**Pull request by crizzitello**
Based on: #141 
Several Things In this PR
 - Use `QStringLiteral` for literal strings
 - Use `tr` for User facing strings
 - Use `clear()` in place of `setText("")` where possible.
 - Use `Q_EMIT` in place of `emit` 
 - Use Const & where possible for ranged loops
 - Add a context to lamda functions.
 - Initialize member variables  
 - Clean up the non-POD static items
 - Let Qt do its normal clean up for `FlowLayout`
 - use `#pragma once` in headers.

### CODE REVIEW

File: src/components/tagging/tageditor.cpp [hunk 1]
```diff
@@ -84,9 +84,9 @@ void TagEditor::wireUi() {
   connect(completer, QOverload<const QString &>::of(&QCompleter::activated), this,
           &TagEditor::completerActivated);
 
-  connect(tagCompleteTextBox, &QLineEdit::textChanged, [this](const QString &text) {
+  connect(tagCompleteTextBox, &QLineEdit::textChanged, this, [this](const QString &text) {
```
Comment (JoelAtDeluxe) [inline src/components/tagging/tageditor.cpp:87]: I don't understand why we're making this change. From what I can tell, the `connect` function takes only 3 parameters when it's connected to a [simple function](https://wiki.qt.io/New_Signal_Slot_Syntax#New:_connecting_to_simple_function). The old code seems to meet that standard. Is this providing some extra functionality, or is it safer, or is this just the preferred style?

Comment (crizzitello) [reply src/components/tagging/tageditor.cpp:87]: This is the safer and preferred connection style for lamda connections.

File: src/forms/evidence_filter/evidencefilter.cpp [hunk 2]
```diff
@@ -35,41 +35,46 @@ QString EvidenceFilters::toString() const {
...
+  static auto triToText = [](Tri t) -> QString {
+      return t == Yes
+              ? QT_TRANSLATE_NOOP("EvidenceFilter", "yes")
+              : QT_TRANSLATE_NOOP("EvidenceFilter", "no");
   };
-  static auto triToText = [](Tri t) -> QString { return t == Yes ? "yes" : "no"; };
 
-  QString rtn = "";
+  QString rtn;
+  auto appendTemp = QStringLiteral(" %1: %2");
```
Comment (JoelAtDeluxe) [inline src/forms/evidence_filter/evidencefilter.cpp:52]: Nice abstraction.

**Review by JoelAtDeluxe — DISMISSED** [review]
Looks good to me.

**Review by jrozner — DISMISSED** [review]

**Review by jrozner — APPROVED** [review]

**Pull request merged**